requirements.txt

```diff
@@ -1,46 +1,12 @@
-awe_languagetool
-awe_spellcorrect
-awe_components
-awe_lexica
-holmes_extractor
-coreferee
-spacy
-spacytextblob <= 3.0.1
-tensorflow
-scipy
-sklearn
-bs4
-rdflib
-jsonpickle
-msgpack-numpy
-falcon
-torch
-wordfreq 
-pygtrie
-symspellpy
-autocorrect
-neuspell
-nltk
-numpy
-aenum
-statistics
-srsly
+awe_languagetool @ git+https://github.com/ArgLab/AWE_LanguageTool.git
```

Comment:
Do we need this file? Everything should probably be in the `setup.cfg` file instead for easier installation.